icepyx/core/read.py

```diff
@@ -684,25 +723,10 @@ def _read_single_grp(self, file, grp_path):
         Xarray dataset with the specified group.
 
         """
-
-        try:
-            grpcat = is2cat.build_catalog(
-                file, self._pattern, self._source_type, grp_paths=grp_path
-            )
-            ds = grpcat[self._source_type].read()
-
-        # NOTE: could also do this with h5py, but then would have to read in each variable in the group separately
-        except ValueError:
-            grpcat = is2cat.build_catalog(
-                file,
-                self._pattern,
-                self._source_type,
-                grp_paths=grp_path,
-                extra_engine_kwargs={"phony_dims": "access"},
-            )
-            ds = grpcat[self._source_type].read()
-
-        return ds
+        # I think this would fail if a group that has too high of a level of nesting
+        # is given. Consider this.
+        # TODO: update docstring
+        return xr.open_dataset(file, group=grp_path)
```

Comment:
For context, this was developed in #222 (which was a monster of a PR to review) that was motivated by #106. And just to be clear, I'm with you on simplifying the code to remove intake now, because the layers of abstraction from intake is getting a bit too hard to read.

Since this `_read_single_grp` method should be about reading a single group, it should be ok? If it's just a one-liner, you could remove the method entirely and call `xr.open_dataset` directly inline. But maybe check what the phony_dims is about, it could be a case that some ATLxx datasets need special handling.

Reply:
> But it feels too easy.

Perhaps I should have made my excitement clearer when you told me how easy it was to read in a group with xarray (and that it would allow us to get rid of intake and thus - I presume - make it possible to read data in from the cloud with fewer changes/using the existing machinery). ;)